python/private/pypi/whl_library.bzl

```diff
@@ -193,6 +193,10 @@ def _whl_library_impl(rctx):
     # Manually construct the PYTHONPATH since we cannot use the toolchain here
     environment = _create_repository_execution_environment(rctx, python_interpreter, logger = logger)
 
+    # Add a PROJECT_ROOT environment variable
+    if rctx.attr.project_root:
+        environment["PROJECT_ROOT"] = str(rctx.path(rctx.attr.project_root).dirname)
```

Comment:
Why do we need an extra value here instead of adding `PROJECT_ROOT` into `envsubst` attr?